tests/gis_tests/distapp/tests.py

```diff
@@ -517,9 +528,7 @@ def test_distance_projected(self):
             dist_qs = [dist1]
         else:
             dist2 = SouthTexasCityFt.objects.annotate(distance=Distance('point', lagrange)).order_by('id')
-            # Using EWKT string parameter.
-            dist3 = SouthTexasCityFt.objects.annotate(distance=Distance('point', lagrange.ewkt)).order_by('id')
```

Comment:
I fixed calling with input for GEOSGeometry in update, though IMO this is overkill.